### PR TITLE
Ensure git is available on the controller.

### PR DESCRIPTION
**Pull request by zancas**
@pjbreaux

### CODE REVIEW

File: playbooks/roles/install_lbaasv2_driver/tasks/main.yml [hunk 1]
```diff
@@ -21,6 +21,11 @@
     state: present
   when: not use_pip|bool
 
+- name: Install git
+  package:
+    name:  '{{ versioned_git_yum_package }}'
```
Comment (pjbreaux) [inline playbooks/roles/install_lbaasv2_driver/tasks/main.yml]: You can make this variable ambiguous, trusting that ansible will do the right thing when it installs git on a debian system (or anything else it supports).

'{{ versioned_git_package }}'

File: playbooks/roles/install_lbaasv2_driver/vars/main.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
+versioned_git_yum_package: git-1.8.3.1-6.el7_2.1
```
Comment (pjbreaux) [inline playbooks/roles/install_lbaasv2_driver/vars/main.yml]: The same variable name change should be made here.

File: playbooks/roles/install_lbaasv2_driver/tasks/main.yml (hunk 1 above)
Comment (zancas) [reply playbooks/roles/install_lbaasv2_driver/tasks/main.yml]: abstracting is cool, but I do want it to be pinned.

**Comment by zancas** [comment]
@pjbreaux testing now...

**Comment by zancas** [comment]
Please see this log for relevant test results:
http://10.190.25.149:49001/job/devs/job/zancas_smoke/5/console

**Pull request merged**